bin/wt

```diff
@@ -47,6 +48,26 @@ cli.addChild(require('./modules'));
 
 Cli.run(cli)
     .timeout(1000 * 60 * 30, Cli.error.timeout('Command timed out after 30 min'))
+    .tap(() => {
```

Comment:
Note: this won't run if there was any sort of error thrown (for example, invalid or missing arguments) so we might want to consider putting this as a `.finally()` block.

Reply:
Do you think this would be a better experience? I actually did not want this to show up when there were errors otherwise as it would be too much of a distraction.